TbsCore/Models/AccModels/AccInfo.cs

```diff
@@ -79,6 +79,11 @@ public void Init()
         /// <summary>
         /// Version of the server account is playing on
         /// </summary>
-        public Classificator.ServerVersionEnum ServerVersion { get; set; }
+        public Classificator.ServerVersionEnum ServerVersion { get; set; }
+
+        /// <summary>
+        /// Is account sitter ?
+        /// </summary>
+        public bool Sitter { get; set; }
```

Comment:
sitter bool should be inside Access, since you can have mutliple access' and one is sitter, another is main. 
https://github.com/Erol444/TravianBotSharp/blob/master/TbsCore/Models/Access/Access.cs
"IsSittering"